misc/SW_WA_2021_2024/guilderton_to_geraldton/breakwalls/README.md

```diff
@@ -0,0 +1,10 @@
+This directory contains a script to create x/y/z data representing the tops of breakwalls.
```

Comment:
It looks like this README needs adjustment to reflect the restructure. It might need:
* Something about first making the breakwalls in subdirectories
* Then info on making the breakwalls list.